src/textures/Texture.js

```diff
@@ -79,6 +79,9 @@ Texture.prototype = {
 
 	copy: function ( source ) {
 
+
+		this.name = source.name + " (Copy)";
```

Comment:
Hmmm... I'm not sure about the `(Copy)` bit. But if we go this way, I guess we should do that with other objects too?